src/index.ts

```diff
@@ -52,9 +52,9 @@ export class MicrosoftStrategy<User> extends OAuth2Strategy<
 
   constructor(
     {
-      clientID,
+      clientId,
```

Comment:
I initially chose to follow the remix-auth conventions with the variable names, but in hindsight, it's probably better to use the Microsoft conventions. 👍